C/include/c4DatabaseTypes.h

```diff
@@ -35,6 +35,7 @@ typedef C4_OPTIONS(uint32_t, C4DatabaseFlags){
         kC4DB_ReadOnly        = 0x02,    ///< Open file read-only
         kC4DB_AutoCompact     = 0x04,    ///< Enable auto-compaction [UNIMPLEMENTED]
         kC4DB_VersionVectors  = 0x08,    ///< Upgrade DB to version vectors instead of rev trees
+        kC4DB_MmapDisabled    = 0x10,    ///< Disable MMAP in SQLite.
```

Comment:
@jianminzhao is this the right code? Should it be something after DiskSyncFull?

Reply:
This is fine to do, we've done it before.  The former keys "bundled" and "shared keys" used to be 0x8 and 0x10.  They were removed years ago and version vectors became the new 0x8.  Mmap can become the new 0x10.

Reply:
Thanks for the clarification 😄